datastore/load.go

```diff
@@ -536,7 +536,7 @@ func propToValue(v *pb.Value) (interface{}, error) {
 	case *pb.Value_DoubleValue:
 		return v.DoubleValue, nil
 	case *pb.Value_TimestampValue:
-		return time.Unix(v.TimestampValue.Seconds, int64(v.TimestampValue.Nanos)), nil
+		return time.Unix(v.TimestampValue.Seconds, int64(v.TimestampValue.Nanos)).In(time.UTC) nil
```

Comment:
Missing a comma here. :smile:

Reply:
Thanks @tbpg If not obvious, I need to author tests/review existing for this yet. I am trying to wrap another task, but also was hoping to have CI let me know how much is left here :) Should get back to this later today.